global/scripts/CorrThick.sh

```diff
@@ -20,41 +20,8 @@ References:
   - https://onlinelibrary.wiley.com/doi/full/10.1002/hbm.25776
 Please cite these works when using this measure.
 
-Requirements:
-  Python dependencies: numpy, nibabel, scipy, psutil
-
-Running stand-alone:
-  To run outside the full HCP pipeline, prepare a batch script that defines at least:
-    - Subject directory
-    - Subject list
-    - Location of the HCP Pipeline environment setup script (SetUpHCPPipeline.sh)
-
-Example batch script:
----------------------------------------
-EnvironmentScript="/path/to/SetUpHCPPipeline.sh"  # HCP Pipeline environment script
-source "$EnvironmentScript"
-
-SubjectDir="/path/to/subjects"
-SubjList="100206 100307 100408 ..."  # space-separated subject IDs
-RegNames="MSMSulc@MSMAll"
-
-for Subject in $SubjList; do
-    "$HCPPIPEDIR/global/scripts/CorrThick.sh" \
-        --subject-dir="$SubjectDir" \
-        --subject="$Subject" \
-        --regnames="$RegNames" \
-        --patch-size=6
-        --skip-computation=NO
-done
-
-OR example usage with fsl_sub:
-
-for Subject in $SubjList; do
-    fsl_sub -q matlabparallel.q "$HCPPIPEDIR/global/scripts/CorrThick.sh" \
-        --subject-dir="$SubjectDir" \
-        --subject="$Subject" \
-done
----------------------------------------'
+# Requirements for this script:
+#  Python3 with dependencies numpy, nibabel, scipy, and psutil'
```

Comment:
If it displays, it shouldn't look like a comment.
```suggestion
This script requires python3 with numpy, nibabel, scipy, and psutil'
```